internal/cluster/transition_test.go

```diff
@@ -4214,6 +4216,69 @@ var _ = Describe("Ocs Operator use-cases", func() {
 			}),
 			errorExpected: false,
 		},
+		{
+			name:               "ocs enabled, 6 nodes, with role of one as auto-assign",
+			srcState:           models.ClusterStatusPendingForInput,
+			dstState:           models.ClusterStatusInsufficient,
+			machineNetworkCidr: "1.2.3.0/24",
+			apiVip:             "1.2.3.5",
+			ingressVip:         "1.2.3.6",
+			dnsDomain:          "test.com",
+			pullSecretSet:      true,
+			hosts: []models.Host{
+				{ID: &hid1, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleAutoAssign},
```

Comment:
Will the list of hosts be returned in the same order in which they were created ? IIf not, how many CPUs/RAM does the default inventory have?

Reply:
Its there in https://github.com/openshift/assisted-service/blob/0372c7aeb81112b92f2c59ee9b2c5e8abfa4170a/internal/cluster/cluster_test.go#L1168